docs/installation_guide.md

```diff
@@ -277,6 +277,47 @@ Please check the [Docker Compose command-line reference](https://docs.docker.com
 By default, registry data is persisted in the target host's `/data/` directory.  This data remains unchanged even when Harbor's containers are removed and/or recreated.
 In addition, Harbor uses `rsyslog` to collect the logs of each container. By default, these log files are stored in the directory `/var/log/harbor/` on the target host.  
 
+## Configuring Harbor listening on a customized port
+By default, Harbor listens on port 80 both for admin portal and docker commands, you can customize the port by following steps:  
+
+1.Modify Deploy/docker-compose.yml  
+Replace the first "80"(HTTP protocal) to a customized port 8888 for example. If HTTPS is needed, replace the first "443".  
```

Comment:
For https, it is unclear whether two ports need to be changed, or only 443 should be changed.
consider something like this if it makes sense:

If HTTPS is enabled, replace the first 443 as well, e.g. 8888:443